common/model/siblings-group.ts

```diff
@@ -0,0 +1,7 @@
+import { Page } from './pages';
+
+export type SiblingsGroup = {
+  id: string;
+  title: string;
+  siblings: Page[];
+};
```

Comment:
Got `js` and `ts` here, any reason why?

Reply:
Assuming to cater for flow? I would prefer not to introduce any more tech debt and use a FlowFixMe.